Added a new annotation class to providing binding to the buttons with…

Keshava11 opened the pull request:
… a defined selector which updates its properties like text, color and background. Added changes in the sample app as well to demo the new binding annotation. not updated appname in library manifest

Keshava11 committed:
Added a new annotation class to providing binding to the buttons with a defined selector which updates its properties like text, color and background. Added changes in the sample app as well to demo the new binding annotation. not updated appname in library manifest

Keshava11 committed:
Added app name in library's manifest. Updated gradle to use lib project instead of dependency

janishar commented:
@Keshava11 Can you please explain the feature you have added in brief so that I can review it better?

Keshava11 commented:
Hi @janishar, I have added a new annotation **BindButtonSelector** which provides binding to the buttons with selector background, selector text color and state dependent text when button toggles between default and selected state.
I have also added it's usage in README.
